webknossos/webknossos/dataset/dataset.py

```diff
@@ -601,11 +601,16 @@ def from_images(
         For more fine-grained control, please create an empty dataset and use
         `add_layer_from_images`.
         """
-        from ._utils.pims_images import get_valid_pims_suffixes
+        from ._utils.pims_images import (
+            get_valid_bioformats_suffixes,
+            get_valid_pims_suffixes,
+        )
 
         input_upath = UPath(input_path)
 
         valid_suffixes = get_valid_pims_suffixes()
+        if use_bioformats is not False:
```

Comment:
If all other PIMs reader fail, do the Libs switch to BioFormats as a last resort? In that case the suffixes need to be added too, right?

Reply:
Yes, but only if `use_bioformats` is `None` or ` True` if it is set explicitly to `False` bioformats is not used by PIMs.